specification/azurefleet/AzureFleet.Management/virtualMachineScaleSet.tsp

```diff
@@ -0,0 +1,81 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using TypeSpec.OpenAPI;
+using Azure.ResourceManager;
+
+namespace Microsoft.AzureFleet;
+
+@doc("An AzureFleet's virtualMachineScaleSet")
+model VirtualMachineScaleSet {
+  @doc("The name of the virtualMachineScaleSet")
+  @key("virtualMachineScaleSetName")
+  @segment("virtualMachineScaleSets")
+  @visibility("read")
+  @path
+  name: string;
+
+  @doc("""
+  The compute RP resource id of the virtualMachineScaleSet 
+  "subscriptions/{subId}/resourceGroups/{rgName}/providers/Microsoft.Compute/virtualMachineScaleSets/{vmssName}"
+  """)
+  @visibility("read")
+  id: string;
```

Comment:
>id [](http://example.com/codeflow?start=2&length=2)

The name, id and type properties will be defined for you by typespec if you use the ProxyResource<> template.

Reply:
See my other comment.

Reply:
Resolved per offline discussion.